### ISSUE TITLE
the wp-sync-media-file plugin does not work with this fork.

### ISSUE THREAD

**Issue by bdlowery** [calvinwinter/wp-database-tools#3]
Any other way to sync media files? Or a way I can manually fix it?

**Comment by robertobz** [comment]
> Any other way to sync media files? Or a way I can manually fix it?

@bdlowery just curious if you found a way to sync media files? thanks

**Comment by bdlowery** [comment]
Nope. Ended up buying the real thing because wasting hours of my time trying to fix this wasn't worth saving $300 a year.
